src/_settings.scss

```diff
@@ -0,0 +1,763 @@
+                           input[type="week"],
+                           input[type="time"],
+                           input[type="date"],
+                           input[type="datetime"],
+                           input[type="datetime-local"],
+                           input[type="number"]' !default;
+
+
+
+
+/* 8. Base (Local)
+   ========================================================================= */
+
+/**
+ * Booleans.
+ */
+
+$scally-base-apply-osx-font-smoothing: false !default;
+
+$scally-base-apply-link-transition: true !default;
+
+$scally-base-apply-style-to-disabled-and-readonly-states: true !default;
+
+$scally-base-apply-bottom-margin-to-paragraphs: false !default;
+
+$scally-base-apply-print-styles: true !default;
+
+
+/**
+ * Root element.
+ */
+
+$scally-base-root-element-background-color: $scally-g-color-white !default;
+
+
+/**
+ * Links.
+ */
+
+$scally-base-link-text-decoration: underline !default;
+
+$scally-base-link-text-decoration-on-hover: none !default;
+
+$scally-base-link-transition-duration: 0.15s !default;
+
+$scally-base-link-transition-timing-function: linear !default;
+
+
+/**
+ * Horizontal rule.
+ */
+
+$scally-base-hr-thickness: 1px !default;
+
+$scally-base-hr-style: solid !default;
+
+$scally-base-hr-color: $scally-g-color-text-base !default;
+
+
+/**
+ * Forms.
+ */
+
+$scally-base-disabled-and-readonly-state-opacity-strength: 0.6 !default;
+
+$scally-base-target-ios-selector: ".ios" !default;
+
+
+
+
+/* 9. Container (Local)
+   ========================================================================= */
+
+/**
+ * Booleans.
+ */
+
+$scally-container-apply-center-alignment: true !default;
+
+
+/**
+ * Widths.
+ */
+
+$scally-container-maximum-width: 1600 !default;
+
+$scally-container-minimum-width: 320 !default;
+
+// Wider
+$scally-container-maximum-width-large: ceil($scally-container-maximum-width * 1.5) !default;
+
+$scally-container-maximum-width-x-large: ceil($scally-container-maximum-width * 2) !default;
+
+// Narrower
+$scally-container-maximum-width-small: floor($scally-container-maximum-width / 1.5) !default;
+
+$scally-container-maximum-width-x-small: floor($scally-container-maximum-width / 2) !default;
+
+
+/**
+ * Gutter size.
+ */
+
+$scally-container-gutter: $scally-g-spacing-x-large !default;
+
+
+
+
+/* 10. Grid (Local)
+   ========================================================================= */
+
+/**
+ * Grid item widths.
+ */
+
+$scally-grid-span-1-width: 8.333% !default;
+
+$scally-grid-span-2-width: 16.666% !default;
+
+$scally-grid-span-3-width: 25% !default;
+
+$scally-grid-span-4-width: 33.333% !default;
+
+$scally-grid-span-5-width: 41.666% !default;
+
+$scally-grid-span-6-width: 50% !default;
+
+$scally-grid-span-7-width: 58.333% !default;
+
+$scally-grid-span-8-width: 66.666% !default;
+
+$scally-grid-span-9-width: 75% !default;
+
+$scally-grid-span-10-width: 83.333% !default;
+
+$scally-grid-span-11-width: 91.666% !default;
```

Comment:
In what way? Looking at it I do wonder if the word `width` needs to be in the setting? Maybe the word `item` should though, so:

```scss
$scally-grid-item-span-1
```